deploy/playbooks/roles/repos/templates/distributions

```diff
@@ -0,0 +1,87 @@
+Codename: sid
+Suite: stable
+Components: main
+Architectures: amd64 armhf i386 source
+Origin: Inktank
+Description: Ceph distributed file system
+DebIndices: Packages Release . .gz .bz2
+DscIndices: Sources Release .gz .bz2
+Contents: .gz .bz2
+SignWith: 17ED316D
```

Comment:
we are not going to be signing, so this should go away otherwise (iirc) reprepro is going to complain that you don't have this key available